x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -149,6 +158,18 @@ export class Simulator {
     return this.domNodes(processNodeElementSelector(options));
   }
 
+  /**
+   * Return an Enzyme ReactWrapper for any child elements of a specific processNodeElement
+   *
+   * @param entityID The entity ID of the proocess node to select in
+   * @param selector The selector for the child element of the process node
+   */
+  public processNodeChildElements(entityID: string, selector: string): ReactWrapper {
```

Comment:
@bkimmel, thoughts? I wanted to generalize your method a bit for any other buttons or things that might be added to the process node in the future

Reply:
:grey_question: It looks good, generally. Two points:
1) The parameter isn't really a `selector` (it only feeds the data-test-subj attrib). So I'd say adjust the comment.
2) To make that even more clear, you _could_ put a "ByDataTestSubj" at the end of the method name (but that's awful long)

Reply:
Yea, we can talk about these patterns later today as well. Gonna merge this for now